kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -298,12 +304,18 @@ def _create_pod_tags_by_pvc(self, pod_list):
             for excluded_tag in self.VOLUME_TAG_KEYS_TO_EXCLUDE:
                 tags = [t for t in tags if not t.startswith(excluded_tag + ':')]
 
-            # get PVC
             for v in volumes:
+                # get PVC
                 pvc_name = v.get('persistentVolumeClaim', {}).get('claimName')
                 if pvc_name:
                     pod_tags_by_pvc['{}/{}'.format(kube_ns, pvc_name)].update(tags)
 
+                # get EVC
+                evc = v.get('ephemeral', {}).get('volumeClaimTemplate')
+                volume_name = v.get('name')
+                if evc and volume_name:
+                    pod_tags_by_pvc['{}/{}-{}'.format(kube_ns, pod_name, volume_name)].update(tags)
```

Comment:
Being `pod_tags_by_pvc` it's unexpected to have `pod_name` in the key. Is there a reason for that? ATM I don't this will be aggregated and I don't see any code that is going to read this key format.

Reply:
When a generic ephemeral volume is created, an associated `pvc` with name `pod_name-volume_name` is created (see https://docs.openshift.com/container-platform/4.11/storage/generic-ephemeral-vols.html#generic-ephemeral-vols-pvc-naming_generic-ephemeral-volumes).
The issue is that this pvc will not have the correct associated tags. I shared some screenshots in the PR's description where you can see that the `pod_name` tag is missing for the standalone PVC.

Reply:
Okay I see, it's worth adding comment in the code explaining that, otherwise the next person will scratch his head wondering why we do that.

Reply:
That's a good point. I just added a comment that explains all of that